pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -1438,7 +1439,17 @@ func validateS3Credentials(data map[string]interface{}, dialer dialer.Dialer) er
 	if err != nil {
 		return err
 	}
-	exists, err := s3Client.BucketExists(context.TODO(), bucket)
+
+	timeout, err := time.ParseDuration(settings.S3BucketCheckTimeout.Get())
+	if err != nil {
+		timeout = 30 * time.Second
+		logrus.Errorf("cluster store: could not parse S3 bucket check timeout, using 30s - error: %v", err)
```

Comment:
Small nit: including `- error` in an error log is a bit redundant, but I won't block over this

Reply:
No problem, I changed this.